From 02753dd2caabfe6b1885cb80a8fb8532b416108d Mon Sep 17 00:00:00 2001
From: Eli Dorfman <elid@voltaire.com>
Date: Tue, 29 Apr 2008 10:12:39 +0300
Subject: [PATCH] IB/iSER: Count fmr alignment violations per session

Count fmr alignment violations per session
as part of the iscsi statistics.

Signed-off-by: Eli Dorfman <elid@voltaire.com>
---
 include/scsi/libiscsi.h |    1 +
 1 file changed, 1 insertion(+)

Index: ofed_kernel/include/scsi/libiscsi.h
===================================================================
--- ofed_kernel.orig/include/scsi/libiscsi.h
+++ ofed_kernel/include/scsi/libiscsi.h
@@ -225,6 +225,7 @@ struct iscsi_conn {
 
 	/* custom statistics */
 	uint32_t		eh_abort_cnt;
+	uint32_t		fmr_unalign_cnt;
 };
 
 struct iscsi_pool {
